templates/web/base/report/new/form_user.html

```diff
@@ -1,17 +1,55 @@
-<div id="private_form">
-<h2 class="form-section-heading form-section-heading--private">[% loc('Private details') %]</h2>
-<p class="form-section-description" id="js-councils_text_private">
-  [% IF js %]
-    [% loc('These will be sent to the council, but will never be shown online.') %]
-    (<a href="/privacy">[% loc('See our privacy policy') %]</a>.)
+<!-- report/new/form_user.html -->
+<div class="js-hide-if-invalid-category">
+
+  [% IF c.user_exists %]
+    <div class="hidden-nojs js-new-report-user-hidden form-section-preview form-section-preview--next">
+      <h2 class="form-section-heading form-section-heading--private hidden-nojs">
+            [% loc('Next:') %] [% loc('Tell us about you') %]
+      </h2>
+      <button id="js-new-report-user-show" class="btn btn--block hidden-nojs">[% loc('Continue') %]</button>
+    </div>
+
   [% ELSE %]
-    [% PROCESS 'report/new/councils_text_private.html' %]
+    <div class="[% 'hidden-nojs' IF NOT c.cobrand.social_auth_enabled %] js-new-report-user-hidden form-section-preview form-section-preview--next">
+      <h2 class="form-section-heading form-section-heading--private hidden-nojs">
+            [% loc('Next:') %] [% loc('Tell us about you') %]
+      </h2>
+      [% IF c.cobrand.social_auth_enabled %]
```

Comment:
With these changes you've made, it looks like new/update form_user.html are basically identical now, so should be factored as one repot/form template somehow :)